test/e2e/gardener/shoot/create_rotate_delete.go

```diff
@@ -450,5 +477,11 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 		Context("Workerless Shoot", Label("workerless"), Ordered, func() {
 			test(NewTestContext().ForShoot(DefaultWorkerlessShoot("e2e-rotate")), false, false)
 		})
+
+		// TODO(AleksandarSavchev): Remove this e2e test when the k8s version for the default shoots is >= 1.34.
+		// For cluseters with version >= 1.34 the single operation rotation is used by `rotate-credentials-start`.
+		Context("Rotate etcd encryption key with single operation", Label("rotate-etcd-encryption-key"), Ordered, func() {
+			testETCDEncryptionKeyRotation(NewTestContext().ForShoot(DefaultShoot("e2e-rot-etcd")))
```

Comment:
What about also adding an e2e test for the etcd key rotation of gardener operator?